tests/test_main.py

```diff
@@ -337,3 +337,55 @@ class Config:
     with pytest.raises(ValueError) as exc_info:
         m.b = 11
     assert '"TestModel" object has no field "b"' in str(exc_info)
+
+
+class ValidateAssignmentModel(BaseModel):
+    a: int = 2
+    b: constr(min_length=1)
+    c: constr(min_length=a)
```

Comment:
This doesn't make sense. Remove `c` completely. 

As per #29 I need to improve how validators work, so I guess it's not really possible to test this case. Sorry.

Reply:
I mean #15 not #29.